README.md

```diff
@@ -11,7 +11,7 @@ dependencies within the same runtime.
 We hope Glazier simplifies using Erlang and CouchDB for you, giving
 a consistent, repeatable build environment.
 
-Of course, you can also use our [previous script collection](README.OLD.md)
+Of course, you can also use our [previous script collection](archive/README.OLD.md)
```

Comment:
Is it truly necessary to keep those old files around?  They will be preserved in the `git` history, that is why I am not sure if we should keep them at all.

Reply:
Yeah I thought about it too, but we should remove them. I will make it clearer in the commit message!